doc/ref/chap-osh-assign.md

```diff
@@ -45,7 +45,7 @@ cleared before starting the modifications.  When the assignment is performed
 with `+=`, the modifications are applied to the existing content of the LHS
 value.
 
-An initializer list has the following form: `(<words>...)`, where each word has
+An initializer list has the following form: `(<items>...)`, where each item has
```

Comment:
I prefer not using `<>` for placeholders,  like `<key>` and `<items>`

That's because bash has so much extra syntax, it's hard to tell if `<>` is part of it or not

Also I use `*` for repetition instead of `...`

So I think it is nicer to write this like

`'(' ITEMS* ')'` like a grammar

And then `[KEY]=VALUE`

and `VALUE`

etc.

Reply:
Thank you. I updated the notation in commit 5c7c4e4c7.